### DISCUSSION TITLE
Dashboard access restriction

### DISCUSSION THREAD

**Discussion by Maduranga** [sorry-cypress/sorry-cypress#513] [Q&A]
I apologize if I have missed this.
We want to use this project in production, but I cannot figure out how to restrict access.
Is there a way to add users?

If not I would like to add OIDC at the AWS ALB, but since this only supports port 80, that is also seems difficult.

Any advice how I can protect access to the dashboard?

Thank you.

**Comment by agoldis** [comment]
Please search existing issues - some people managed to enable restricted access, it isn't supported out of the box, though
